dist/angular-numbro.min.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * AngularJS filter for Numbro.js: number formatting as a filter
+ * @version v1.1.6 - 2014-10-29
+ * @link https://github.com/baumandm/angular-numbro
+ * @author Dave Bauman <baumandm@gmail.com>
+ * @license MIT License, http://www.opensource.org/licenses/MIT
+ */
+"use strict";angular.module("ngNumbro",[]).provider("$numbroConfig",function(){var a={};this.setFormat=function(b,c){a[b]=c},this.setDefaultFormat=function(a){numbro.defaultFormat(a)},this.setLanguage=function(a,b){numbro.language(a,b)},this.setCurrentLanguage=function(a){numbro.language(a)},this.$get=function(){return{customFormat:function(b){return a[b]||b}}}}).filter("numbro",["$numbroConfig",function(a){return function(b,c){return null==b?b:(c=a.customFormat(c),numbro(b).format(c))}}]);
```

Comment:
Line is too long.<br>Use the function form of "use strict".<br>Missing semicolon.<br>Expected an assignment or function call and instead saw an expression.<br>Use '===' to compare with 'null'.<br>'numbro' is not defined.